x/bank/types/msgs.go

```diff
@@ -79,6 +79,10 @@ func (msg MsgMultiSend) ValidateBasic() error {
 		return ErrNoInputs
 	}
 
+	if len(msg.Inputs) != 1 {
+		return ErrMultipleSenders
```

Comment:
we should add a state machine breaking changelog.

We should also bump the bank module's consensus version